gxp_21_cfr_part_11/11_200.sp

```diff
@@ -1,8 +1,18 @@
 benchmark "gxp_21_cfr_part_11_11_200" {
   title       = "11.200 Electronic signature components and controls"
+  description = "TO DO"
```

Comment:
What is this supposed to be?

Reply:
Do we have any description we can use? Will the Hub page look blank/empty (assming we don't have a separate doc for this one)?